### PR TITLE
Add BSS logo

### PR DESCRIPTION
**Pull request by candrasaputra**
## Why are you adding this icons?
We provide a [Shopify Payment App](https://apps.shopify.com/payments-by-xendit) and need to show the logo in the checkout page.

#### Help us identify yourself
- [x] I'm working/collaborating with the brand directly and they have provided the icons.
- [ ] I'm associated with the brand and I've read all the brand icon’s guidelines.
- [ ] I'm an individual and I've read all the brand icon’s guidelines.

Link to the brand guidelines: <Link>

## Checklist to add new icons
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.
- [x] I recognize that if my icon is not approved by the Shopify Partners team it may not receive review nor merger.

If this pull request is not adding new icons, you can remove this checklist.

## Attach a screenshot of the icon with the black background
Screen Shot 2022-10-11 at 07 33 20

### CODE REVIEW

**Review by yanoandri — APPROVED** [review]

**Comment by candrasaputra** [comment]
hi @rahilvirani please help to review

**Comment by candrasaputra** [comment]
hi @rahilvirani please help to review

**Review by Jacquesattaque — APPROVED** [review]

File: db/payment_icons.yml [hunk 1]
```diff
@@ -1549,7 +1553,9 @@
 -  
   name: spei
   label: SPEI
-  group: bank_transfers
+  group: 
+  
+
```
Comment (lunataoo) [inline db/payment_icons.yml:1558]: Please fill in this field, TY

**Review by lunataoo — CHANGES_REQUESTED** [review]

**Comment by Jacquesattaque** [comment]
@candrasaputra Please let us know if you still require this icon. If we haven't heard back by the end of next week we'll close this issue.

**Comment by candrasaputra** [comment]
@Jacquesattaque Hi, yes we still need this.
let me fix the issue.

**Comment by andykim** [comment]
Hi @Jacquesattaque . I fixed the issue. Can you please help to review it again? Thanks :pray:

**Review by lunataoo — APPROVED** [review]

**Pull request merged**